frontend/src/components/1.Landing/StartPage.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {useState} from 'react';
```

Comment:
Something really nitpicky, but we could combine these two lines for each page file
`import React, {useState} from 'react'`